lib/vector-core/src/enrichment_table/enrichment_tables.rs

```diff
@@ -0,0 +1,286 @@
+use super::EnrichmentTable;
+use arc_swap::ArcSwap;
+#[cfg(feature = "vrl")]
```

Comment:
Random thought: do we want this to live in `vector-core`, or should the actual integration between Vector and VRL live in `vector` itself?

I guess we also have the `vector::Value` <-> `vrl::Value` interop live in core, so I guess that answers my own question?

Reply:
I'll defer to others on the final decision here, but my thinking is in line with your conclusion that it lives alongside `vrl::Value`.